FordLabs/retroquest
# 404 page icons

## What was done
- Update the 404 page's icon to be a font-awesome icon instead of an svg
- Update the 404 page icon to change to be one of 4 icons every time the page loads

### Demo
<img width="491" alt="Screen Shot 2022-11-21 at 2 31 45 PM" src="https://user-images.githubusercontent.com/17144844/203142800-25d8f616-a046-45f0-bd28-106459f85c43.png">
<img width="554" alt="Screen Shot 2022-11-21 at 2 31 37 PM" src="https://user-images.githubusercontent.com/17144844/203142809-bc85c7f2-efe5-45da-b595-ec13eba4d059.png">
<img width="562" alt="Screen Shot 2022-11-21 at 2 31 25 PM" src="https://user-images.githubusercontent.com/17144844/203142834-b7072769-6933-4f14-855d-2bfb079c3f17.png">
<img width="476" alt="Screen Shot 2022-11-21 at 2 32 27 PM" src="https://user-images.githubusercontent.com/17144844/203142934-58e4ab96-1609-4bc1-95b2-d8c7e038478e.png">

## Testing Instructions
1) Ensure the 404 page icon is random on each page load
2) Ensure the color of the icon matches that of the text in both light and dark mode, and in various browsers.
